src/components/grid-aware/PartnersAndSponsorsBlock/PartnersAndSponsorsBlock.module.css

```diff
@@ -0,0 +1,63 @@
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: repeat(2, auto);
```

Comment:
It looks like you only have one row, so you don't need to declare this, which declares that there are two rows. Omitting this property altogether will just create a single, auto-sized row.